themes/bootstrap3/templates/search/list-grid.phtml

```diff
@@ -1,4 +1,17 @@
-<div class="<?=$this->escapeHtmlAttr(trim('search-grid ' . ($this->resultsClass ?? '')))?>">
+<?php
+  $listClasses = ['search-grid'];
+  if ($this->results->getOptions()->loadResultsWithJsEnabled()) {
+    $this->headScript()->appendFile('search.js');
+    // Ensure OpenURL support is available even if none of the results on the initial
+    // page has one:
+    $this->headScript()->appendFile('openurl.js');
+    $listClasses[] = 'js-record-list';
+  }
+  if ($class = $this->resultsClass ?? '') {
+    $listClasses[] = $this->escapeHtmlAttr($class);
```

Comment:
If changes are made to list-list.phtml, they'll need to be matched here. I wonder if there's a way to avoid redundancy between these two templates (though that doesn't seem likely without a view helper, which is probably overkill).

Reply:
This was already the case before, just not as prominent. Maybe cross-linked comments would help?

Reply:
I did some refactoring to reduce duplication here as well as in GetSearchResults AJAX handler by adding results-list.phtml template between results.phtml and list-*.phtml (kind of like what combined search has, but just for setting classes and other attributes).